README.md

```diff
@@ -1,38 +1,114 @@
 Minecraft Server Control Script
-=============================
+===============================
 
 A powerful command-line control script for UNIX and Linux powered Minecraft servers.
 
-
-## Index
-* [Features](#features)
+# Index
+* [Overview](#overview)
+* [Prerequisites for installation](#prerequisites-for-installation)
+  * [Required programs](#required-programs)
+  * [Configuring the firewall / NAT](#configuring-the-firewall--nat)
+  * [Mapping software (optional)](#mapping-software-optional)
 * [Installation](#installation)
-* [Usage](#usage)
-* [Server Customization](#server-customization)
+  * [Downloading the script](#downloading-the-script)
+  * [Configuration](#configuration)
+* [Creating and importing worlds](#creating-and-importing-worlds)
+  * [Create new world](#create-new-world)
+  * [Import existing world](#import-existing-world)
+* [Adjusting server options](#adjusting-server-options)
+  * [The mscs.properties file](#the-mscs.properties-file)
+  * [The msctl file](#the-msctl-file)
+* [Automated backups and restarts](#automated-backups-and-restarts)
+  * [Scheduling backups](#scheduling-backups)
+  * [Removing backups after X days](#removing-backups-after-x-days)
+  * [Scheduling restarts](#scheduling-restarts)
+* [Mapping the world](#mapping-the-world)
+* [Command reference](#command-reference)
+  * [Examples](#examples)
 * [License](LICENSE)
 * [Issues](#issues)
 
+## Overview
+**M**inecraft **S**erver **C**ontrol **S**cript (**MSCS**) is a server-management script for UNIX and Linux powered Minecraft servers. 
+Features include:
 
-## Features
 * Run multiple Minecraft worlds.
 * Start, stop, and restart single or multiple worlds.
 * Create, delete, disable, and enable worlds.
 * Includes support for additional server types: [Forge](http://www.minecraftforge.net/), [BungeeCord](http://www.spigotmc.org/wiki/bungeecord/), [SpigotMC](http://www.spigotmc.org/wiki/spigot/), etc.
 * Users automatically notified of important server events.
-* Uses the Minecraft [Query protocol](http://wiki.vg/Query) to keep track of current server conditions.
+* Uses the Minecraft [Query protocol](http:b //wiki.vg/Query) to keep track of current server conditions.
 * LSB and systemd compatible init script, allows for seamless integration with your server's startup and shutdown sequences.
 * Map worlds using the [Minecraft Overviewer](http://overviewer.org/) mapping software.
-* Backup worlds, and remove backups older than X days.
+* Automatically backup worlds, remove backups older than X days, and restart worlds.
 * Update the server and client software automatically.
 * Send commands to a world server from the command line.
 
-See the [Usage](#usage) section below for a description on how to use these features.
+Most of the general/beginner documentation is located in this file. For additional documentation that is not located in this file, please check the [wiki](https://github.com/Roflicide/MinecraftServerControlScript/wiki). 
```

Comment:
Hi Michael, thanks for the patches to the README.

> 1. msctl file

> I don't quite understand why it's not a good idea to edit this file. I saw that there were options for settings in this file, so why do I need to edit a file elsewhere? I also tried looking for the files above in all of the paths you listed above but I couldn't seem to find any of the mscs.conf files. I also modified this file directly and it seemed to work fine for my server. 

While it works just fine to edit the script files, if you want to upgrade to the latest version that I release you are going to have a hard time moving all your changes over.  It is just easier to place local changes in the `/etc/default/mscs` file.

> 2. location of msctl

> I guess I'm confused on the difference between msctl and mscs. I thought that, since the command was "mscs", that the mscs file would be the one I wanted to point towards backups. Why do I need to point it towards the msctl file? I pointed it towards the mscs file with my server and it's working just fine.

The `mscs` script is the one that you probably want to be using and referencing in the README, it is the script that should be used for a normal system-wide install.  The `msctl` script is essentially undocumented (see Issue #102), but it is meant to be used in a multi-user environment where each user is able to control their own servers.

> 3. git pull vs git clone

> Why should it not be used? I don't understand why git pull should be used instead of git clone.

`git clone` is used to make a new local copy of a repository, `git pull` is used to update a local copy to match the copy in the repository.  The README file should probably mention `git clone`, not `git pull`.  You may be able to abuse one or the other for other purposes. ;)